src/Components/Web.JS/src/Services/NavigationEnhancement.ts

```diff
@@ -116,13 +119,26 @@ function onDocumentSubmit(event: SubmitEvent) {
       return;
     }
 
+    const submitter = event.submitter as HTMLButtonElement;
+
+    if ((submitter && submitter.formMethod && submitter.formMethod === 'dialog') ||
+      formElem.method === 'dialog') {
+      logger.log(LogLevel.Warning, 'Form cannot be enhanced when method = "dialog".');
+      return;
+    }
+
+    if ((submitter && submitter.formTarget && submitter.formTarget !== '_self') ||
+      formElem.target !== '_self') {
```

Comment:
```suggestion
    if (submitter?.formTarget !== '_self' || formElem.target !== '_self') {
```

Reply:
Thank you for the suggestion!
I did not apply it because in case when there is no formtarget the first part `submitter?.formTarget !== '_self'` equals true. That is not the result I want. 
I changed it to `submitter?.formTarget ?? formElem.target) !== '_self'` instead.

Reply:
Thank you for the suggestion!
I didn't apply it because when there is no `formtarget` then first part `submitter?.formTarget !== '_self'` equals true. 
Also, I found that `target` is not specified then `formElem.target` equals empty string.